daemon/src/model.rs

```diff
@@ -789,7 +764,16 @@ impl FeeAccount {
     }
 
     pub fn add_opening_fee(self, opening_fee: OpeningFee) -> Self {
-        let sum = self.balance + opening_fee.into_signed_amount(self.role);
+        let fee: i64 = opening_fee.fee.as_sat().try_into().expect("not to overflow");
+
+        let signed_fee = match self.role {
+            Role::Maker => -fee,
+            Role::Taker => fee,
+        };
+
+        let signed_fee = SignedAmount::from_sat(signed_fee);
```

Comment:
To remove this boilerplate, you could add an extension trait for `Amount` that has an `invert` method and returns us a `SignedAmount`.

Reply:
we actually had that last week!

Reply:
Happy to see this in a follow-up